store/tools/ics23-iavl/.circleci/config.yml

```diff
@@ -0,0 +1,22 @@
+# This is a bit complex with multiple jobs.
+# https://circleci.com/docs/2.0/local-cli/ will let you test this locally
+version: 2
+jobs:
```

Comment:
I think we need to convert it into github CI, so moving to `.github` directory (probably toplevel) and change images.

Reply:
The circleci configs were pretty trivial, so I think the existing CI will cover tests already